tex/artikel/Bachelor_Master.tex

```diff
@@ -1,15 +1,18 @@
-% Dieser Artikel kann in ein paar Jahren weg, wenn sich keiner der Erstis mehr
-% an Bologna/das Diplom erinnert ;-)
-\section{Von den Anfängen des Bachelor-Master-Systems}
+% In 2021 umgeschrieben, da sich kaum einer mehr ans Diplom erinnert. Fokus jetzt mehr auf Ba-Ma-System anstatt Historie.
+\section{Das Bachelor-Master-System im Schnelldurchlauf}
 
 \begin{multicols}{2}
 \begin{quote}
```

Comment:
Dieses Zitat kann auch entfernt werden, wenn der Text kürzer werden soll. Aktuell sind es 7 Zeilen weniger als in der vorherigen Version.